docs/source/development/style-guide.rst

```diff
@@ -0,0 +1,392 @@
+``imap_processing`` Style Guide
+=============================
+
+This document serves as a style guide for all ``imap_processing`` software development. Any potential contribution to
```

Comment:
Add a link to the repository? May be overkill. Also, if this applies to sds-data-manager (I think it should) then add that here too

Reply:
Good idea. Now that our docs are being served in a more central location, we can apply this to both repos.